index.html

```diff
@@ -1,576 +1,589 @@
+        This proposed charter is available on <a href="https://github.com/w3c/rch-wg-charter">GitHub</a>.
+        Feel free to raise <a href="https://github.com/w3c/rch-wg-charter/issues">issues</a>.
+      </p>
+
+      <div id="details">
+        <table class="summary-table">
+          <tr id="Status">
+            <th>
+              Charter Status
+            </th>
+            <td>
+              <i class="todo">See the <a href="https://www.w3.org/groups/wg/rch/charters/">group status page</A> and <a href="#history">detailed change history</a>.</i>
+            </td>
+          </tr>
+          <tr id="Duration">
+            <th>
+              Start date
+            </th>
+            <td>
+              <i class="todo">[dd monthname yyyy] (date of the "Call for Participation", when the charter is approved)</i>
+            </td>
+          </tr>
+          <tr id="CharterEnd">
+            <th>
+              End date
+            </th>
+            <td>
+              <i class="todo">[dd monthname yyyy]</i> (Start date + 2 years)
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Chairs
+            </th>
+            <td>
+              <a href="mailto:phil.archer@gs1.org">Phil Archer</a>, <a href="https://www.gs1.org/">GS1</a><br>
+              <a href="mailto:markus@danubetech.com">Markus Sabadello</a>, <a href="https://danubetech.com">Danube Tech</a>
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Team Contacts
+            </th>
+            <td>
+              <a href="mailto:pierre-antoine@w3.org">Pierre-Antoine Champin</a> (0.05 <abbr title="Full-Time Equivalent">FTE</abbr>)
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Meeting Schedule
+            </th>
+            <td>
+              <strong>Teleconferences:</strong> topic-specific calls will be held on a per need basis. <br> <strong>Face-to-face:</strong> face-to-face meetings may be scheduled by consent of the participants, usually no more than 3 per year.
+            </td>
+          </tr>
+        </table>
+
+      </div>
+
+      <div id="background" class="background">
+        <h2>Motivation and Background</h2>
+        <p>
+          There are a variety of use cases that depend on the ability to calculate a unique and deterministic hash value of RDF Datasets, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. See the <a href="explainer.html#usage">use cases</a> for more examples. These use cases require a standard way to process the underlying graphs contained in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```

Comment:
```suggestion
          There are a variety of use cases that depend on the ability to calculate a unique and deterministic hash value of RDF Datasets, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. See the <a href="https://www.w3.org/TR/rch-explainer/#usage">use cases</a> for more examples. These use cases require a standard way to process the underlying graphs contained in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```
Similar to @philarcher 's suggestion.